tests/unit/workflows/test_workflows_tasks_arxiv.py

```diff
@@ -152,3 +158,30 @@ def test_arxiv_derive_inspire_categories_does_nothing_with_existing_categories()
 
     assert validate(result, inspire_categories_schema) is None
     assert expected == result
+
+
+@patch('inspirehep.modules.workflows.tasks.arxiv.os')
+def test_arxiv_author_list_handles_auto_ignore_from_arxiv(mock_os):
+    data = {
+        'arxiv_eprints': {
+            'value': ['1703.09986']
+        }
+    }
+    mock_arxiv_tarball = MockFiles({
+        '1703.09986.tar.gz': AttrDict({
+            'file': AttrDict({
+                'uri': resource_filename(__name__, 'fixtures/1703.09986.tar.gz')
```

Comment:
Extremely small nit: `fixtures/1703.09986.tar.gz` uses the UNIX separator, which is not OS independent. So we can use `os.path.join` to use `os.path.sep` instead.

I'm actually not sure if `pkg_resources` handles this for us, but I don't have a Windows system to test this incredibly small detail.